### DISCUSSION TITLE
Adding new features to existing MinimalExtractionParameters settings

### DISCUSSION THREAD

**Discussion by utkarshtri1997** [blue-yonder/tsfresh#923] [Q&A]
Hi, Firstly Thank you for building this good application for time-series calculation. Can you please help me in adding new features to MinimalExtractionParameters? Please reply ASAP.`@set_property("fctype", "simple")
def flor_ratio(x):
"""
The description of your feature

:param x: the time series to calculate the feature of
:type x: pandas.Series
:return: the value of this feature
:return type: bool, int or float
"""
# Calculation of feature as float, int or bool
u=np.max(x)
v=np.min(x)
#flor_ratio = (u-v)/v
return (u-v)/v
fc_parameters = {
'median': None,
'mean': None,
'standard_deviation': None,
'root_mean_square': None,
'maximum': None,
'minimum': None,'flor_ratio':None
from sktime.transformations.panel.tsfresh import TSFreshRelevantFeatureExtractor
from tsfresh.feature_extraction import extract_features
transformer = TSFreshRelevantFeatureExtractor(kind_to_fc_parameters= fc_parameters)
extracted_features = transformer.fit_transform(X_train,y_train)
extracted_features
}` But this doesn't calculate the features I gave it to calculate.

**Comment by kempa-liehr** [comment]
Hi,
Please have a look at the documentation: https://tsfresh.readthedocs.io/en/latest/text/how_to_add_custom_feature.html
Cheers,
Andreas

**Reply by utkarshtri1997** [reply]
Thank you for your reply. I have read that document previously. You can see the code I have posted. I did define the kind_to_fc_parameters equal to fc_parameters where I have included my new feature name and its parameter(None). I am using it in the sktime and calling TSFreshRelevantFeatureExtractor/TSFreshFeatureExtractor. It extracts the feature but not the features I gave it to. It calculates 300 features while I am giving 7 features as parameters. 
Please look into the problem. I can share more parts of the code if needed.

**Comment by utkarshtri1997** [comment]
Is anybody else going to answer this?
